src/Entity/Aspect.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\AspectRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=AspectRepository::class)
+ * @ORM\Table(name="quap_aspect")
+ */
+class Aspect
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $name_de;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $name_fr;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $name_it;
+
+    /**
+     * @ORM\OneToMany(targetEntity = "Question", mappedBy = "aspect")
+     * @ORM\JoinColumn
+     */
+    private $question;
+
+    /**
+     * @ORM\ManyToOne(targetEntity = "Questionnaire", inversedBy = "aspect")
+     * @ORM\JoinColumn
+     */
+    private $questionnaire;
+
+    public function __construct()
+    {
+        $this->questionnaire = new ArrayCollection();
```

Comment:
The questionnaire is no array collection! I think you meant to set the $question to an array collection